examples/js/exporters/GLTFExporter.js

```diff
@@ -800,9 +800,41 @@ THREE.GLTFExporter.prototype = {
 
 					for ( var attributeName in geometry.morphAttributes ) {
 
+						// glTF 2.0 morph supports only POSITION/NORMAL/TANGENT.
+						//
+						// @TODO TANGENT support
```

Comment:
TBH I wouldn't bother adding a @\TODO about this since three.js doesn't support stored tangents currently...